pages/embed.tsx

```diff
@@ -241,7 +241,7 @@ const EmbedPage = ({ config, data, error }: IEmbedPageProps) => {
   }, [handleResize]);
 
   if (error) {
-    return <Error statusCode={error.statusCode} title={error.message} />;
```

Comment:
This title needs to stay. It is not the same as an HTML title attribute. It is the property that this component uses to provide the error message.